ansible/roles/proteus-orchestrate/templates/proteus-orchestrate.toml.j2

```diff
@@ -13,13 +13,7 @@ gorush-basic-auth-password = "{{ proteus_notify_basic_auth_password }}"
 [api]
 port = {{ proteus_api_port }}
 address = "{{ proteus_api_address }}"
+default-inputs-to-return = 10
```

Comment:
I don't remember reading about this key, but I assume it's implemented.

Reply:
I should actually probably drop this from the default config and here, as I removed from the source since I don't think it makes sense to have config defined defaults (if it's a default value for something it should not be overridable from the config...)

Reply:
> I don't think it makes sense to have config defined defaults

Sounds very correct to me. Please zap that!